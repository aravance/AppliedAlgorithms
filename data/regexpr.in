The Unix* system has become quite popular since its inception
in 1969, running on    machines of varying
processing power from microprcessors to mainframes
and providing a common
execution
environment across them.
*
/ a/
/on/
/on /
/o. /
/o. r/
/o. m/
/o. *m/
/\*/
*
