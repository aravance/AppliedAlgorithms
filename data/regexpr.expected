The Unix* system has become quite popular since its inception
in 1969, running on    machines of varying
processing power from microprcessors to mainframes
and providing a common
execution
environment across them.

/ a/
and providing a common
environment across them.

/on/
The Unix* system has become quite popular since its inception
in 1969, running on    machines of varying
and providing a common
execution
environment across them.

/on /
in 1969, running on    machines of varying

/o. /
in 1969, running on    machines of varying
processing power from microprcessors to mainframes

/o. r/
<no lines>

/o. m/
processing power from microprcessors to mainframes

/o. *m/
in 1969, running on    machines of varying
processing power from microprcessors to mainframes

/\*/
The Unix* system has become quite popular since its inception

